README.md

```diff
@@ -5423,6 +5423,7 @@ This notification tells an app to update the AddSubMenu or its 'sub' AddCommand
 ### EncodedSyncPData
 Message Type: **request**
 
+deprecated since 7.1.0
```

Comment:
Probably would have to be done in its own PR, but for formatting purposes, there should be a newline added after this.
```
-                            mark_down.write('\ndeprecated since ' + item.since)
+                            mark_down.write('\ndeprecated since {}\n'.format(item.since))
```

Reply:
@jacobkeeler agreed. Created a separate PR for the formatting fix https://github.com/smartdevicelink/rpc_spec/pull/312